lib/formatters/table.js

```diff
@@ -0,0 +1,134 @@
+"use strict";
```

Comment:
Please use our standard header and follow our conventions for style and comments: http://eslint.org/docs/developer-guide/code-conventions.html